client/blocks/subscription-length-picker/option.jsx

```diff
@@ -0,0 +1,126 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { uniqueId } from 'lodash';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+
+import Badge from 'components/badge';
+import { TERM_ANNUALLY, TERM_BIENNIALLY, TERM_MONTHLY } from 'lib/plans/constants';
+
+export class SubscriptionLengthOption extends React.Component {
+	static propTypes = {
+		term: PropTypes.string.isRequired,
+		savePercent: PropTypes.number.isRequired,
+		price: PropTypes.string.isRequired,
+		pricePerMonth: PropTypes.string.isRequired,
+		checked: PropTypes.bool.isRequired,
+		value: PropTypes.any.isRequired,
+		onCheck: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		checked: false,
```

Comment:
docs don't mention that this is optional, and `propTypes` shows it as required, but we supply a default value here, which seems contradictory

Reply:
addressed